index.html

```diff
@@ -59,11 +59,11 @@ <h1 class="intro_2">
       </div>
 
       <ul class="social_media_icons">
-        <li><a href="https://www.linkedin.com/in/bonke-gcobo-28a763125/"><img class="head_img" src="images/twitter.png" alt="twitter_bird" /></a></li>
-        <li><a href="https://www.linkedin.com/in/bonke-gcobo-28a763125/"><img class="head_img" src="images/M_icon.jpg" alt="M_picture" /></a></li>
+        <li><a href="https://twitter.com/home"><img class="head_img" src="images/twitter.png" alt="twitter_bird" /></a></li>
+        <li><a href="https://analytics.google.com/analytics/web/#/provision"><img class="head_img" src="images/M_icon.jpg" alt="M_picture" /></a></li>
```

Comment:
- [ ] A new issue appears, now your mobile version has an unexpected horizontal scrollbar, kindly fix it.

![image](https://user-images.githubusercontent.com/60164239/131693627-afacb7fc-862c-480a-81e0-bd7d9019a594.png)